### PR TITLE
Update selectors and URL

### PR DESCRIPTION
**Pull request by iamsameeraliyanage**
Update selectors and URL

# Pull request motivation(s)

### What is the current behaviour?

*If the current behaviour is a bug, please provide all the steps to reproduce and screenshots with context.*

##### NB2: Any other feedback / questions ?
I was asked previously I wanted "**[https://lib.ballerina.io](https://lib.ballerina.io)**" domain should be in the search result if i search "**SwanLake**". You helped me with that. Thanks for that,
Now its working partially.  What I mean that, If I search **Swanlake API** Search result is there. But if I click the link it goes to inner page([https://lib.ballerina.io/ballerina/udp/0.9.0-beta.2#main](https://lib.ballerina.io/ballerina/udp/0.9.0-beta.2#main)) not to the home page(**[https://lib.ballerina.io](https://lib.ballerina.io)**).
Can you help with that?

### CODE REVIEW

**Comment by iamsameeraliyanage** [comment]
I was asked previously I wanted "https://lib.ballerina.io" domain should be in the search result if i search "SwanLake". You helped me with that. Thanks for that,
Now its working partially. What I mean that, If I search Swanlake API Search result is there. But if I click the link it goes to inner page(https://lib.ballerina.io/ballerina/udp/0.9.0-beta.2#main) not to the home page(https://lib.ballerina.io).
Can you help with that?

**Comment by shortcuts** [comment]
Hey @iamsameeraliyanage,

Unfortunately, links correspond to where the content has been crawled, so it won't be possible to do this from the backend, unless you add/edit a page on your side for this purpose (or with a paying plan that let's you add custom records directly from the Algolia API or the Dashboard to your index)

Searching for `The following is a list of language and standard library modules that are available with the distribution.` for example, will redirect you to the homepage since that's where the record has been crawled.

File: configs/ballerina.json [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
       },
       "page_rank": 1
     },
     {
-      "url": "https://lib.ballerina.io/",
+      "url": "https://lib.ballerina.io",
       "page_rank": 2,
       "selectors_key": "swanlake_api_docs",
-      "tags": ["swanlake_api_docs","api","swanlake"],
+      "tags": ["swanlake_api_docs","api","swanlake","swan","lake"],
```
Comment (shortcuts) [inline configs/ballerina.json:16]:
```suggestion
      "tags": ["swanlake_api_docs"],
```

Tags are added for filtering purposed, unless you have a use for all of these, you can restrict to `swanlake_api_docs` for example.

See https://docsearch.algolia.com/docs/config-file/#using-custom-tags

**Comment by iamsameeraliyanage** [comment]
Hi Clément Vannicatte,
Thank you. I get your point. 
Thank you for the respond. Please merge this.

**Pull request merged**